sdk/keyvault/azkeys/models.go

```diff
@@ -3,457 +3,563 @@
-	RecoveryLevel *string
+// ListDeletedKeysOptions contains the optional parameters for the Client.ListDeletedKeys method.
+type ListDeletedKeysOptions struct {
+	// Maximum number of results to return in a page. If not specified the service will return up to 25 results.
+	MaxResults *int32
+}
+
+// ListKeyVersionsOptions contains the optional parameters for the Client.ListKeyVersions method.
+type ListKeyVersionsOptions struct {
+	// Maximum number of results to return in a page. If not specified the service will return up to 25 results.
+	MaxResults *int32
+}
+
+// ListKeysOptions contains the optional parameters for the Client.ListKeys method.
+type ListKeysOptions struct {
+	// Maximum number of results to return in a page. If not specified the service will return up to 25 results.
+	MaxResults *int32
+}
+
+// PurgeDeletedKeyOptions contains the optional parameters for the Client.PurgeDeletedKey method.
+type PurgeDeletedKeyOptions struct {
+	// placeholder for future optional parameters
+}
+
+// RecoverDeletedKeyOptions contains the optional parameters for the Client.RecoverDeletedKey method.
+type RecoverDeletedKeyOptions struct {
+	// placeholder for future optional parameters
+}
+
+// ReleaseOptions contains the optional parameters for the Client.Release method.
+type ReleaseOptions struct {
+	// placeholder for future optional parameters
+}
+
+// RestoreKeyOptions contains the optional parameters for the Client.RestoreKey method.
+type RestoreKeyOptions struct {
+	// placeholder for future optional parameters
+}
+
+// RotateKeyOptions contains the optional parameters for the Client.RotateKey method.
+type RotateKeyOptions struct {
+	// placeholder for future optional parameters
+}
+
+// SignOptions contains the optional parameters for the Client.Sign method.
+type SignOptions struct {
+	// placeholder for future optional parameters
+}
+
+// UnwrapKeyOptions contains the optional parameters for the Client.UnwrapKey method.
+type UnwrapKeyOptions struct {
+	// placeholder for future optional parameters
+}
+
+// UpdateKeyOptions contains the optional parameters for the Client.UpdateKey method.
+type UpdateKeyOptions struct {
+	// placeholder for future optional parameters
+}
+
+// UpdateKeyRotationPolicyOptions contains the optional parameters for the Client.UpdateKeyRotationPolicy method.
+type UpdateKeyRotationPolicyOptions struct {
+	// placeholder for future optional parameters
+}
+
+// VerifyOptions contains the optional parameters for the Client.Verify method.
+type VerifyOptions struct {
+	// placeholder for future optional parameters
+}
+
+// WrapKeyOptions contains the optional parameters for the Client.WrapKey method.
+type WrapKeyOptions struct {
+	// placeholder for future optional parameters
+}
+
+// CreateKeyParameters - The key create parameters.
+type CreateKeyParameters struct {
+	// REQUIRED; The type of key to create. For valid values, see JsonWebKeyType.
+	Kty *JSONWebKeyType `json:"kty,omitempty"`
```

Comment:
I think we should keep this one although it could seem odd to users who don't know the context. "Kty" comes from the JSON Web Key RFC. Key Vault represents keys as JWKs, so it makes some sense to use "kty" consistently throughout the API, beyond the `JSONWebKey` model.